src/OrchardCore.Modules/OrchardCore.ContentLocalization/Sitemaps/LocalizedContentItemsQueryProvider.cs

```diff
@@ -70,7 +70,7 @@ public async Task GetContentItemsAsync(ContentTypesSitemapSource source, Content
                     var queryResults = await _session.Query<ContentItem>()
                          .With<ContentItemIndex>(ci => ci.ContentType == source.LimitedContentType.ContentTypeName && ci.Published)
                          .OrderBy(ci => ci.CreatedUtc)
-                         .With<LocalizedContentItemIndex>()
+                         .With<LocalizedContentItemIndex>(i => i.Published || i.Latest)
```

Comment:
Think it should only be Published here, draft items shouldn't in the sitemap.
Although they should already be excluded by the `ContentItemIndex` part of the query.

So an extra predicate here might not be necessary at all?

Reply:
See my next comment for why i added this globaly

But yes, here seems useless as above there is `ci.Published` on the `ContentItemIndex`

Good catch, i will remove it